src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -112,20 +116,69 @@ export const ARShowScreen = ({ navigation, route }) => {
   );
 };
 
+/**
+ * index creation function for model and texture
+ *
+ * @param {string} startDate           start date of the model
+ * @param {number} timePeriodInDays    time period in days of the model
+ * @param {array}  scenes              array of models
+ *
+ * @return {object} both parsed values as an object, like { modalIndex: 1, textureIndex: 1 }
+ */
+const multipleSceneIndexGenerator = async ({ startDate, timePeriodInDays, scenes }) =>
```

Comment:
please move everything but the component and styles into other files. the screen is too bloated.
and as i already mentioned once, having logics under the component in the same file is bad practice.

Reply:
Yeah, you're right, I will. 👍